### DISCUSSION TITLE
Pyinstaller Linux

### DISCUSSION THREAD

**Discussion by ThorTomas** [TomSchimansky/CustomTkinter#1604] [General]
I am struggling with converting my app with customtkinter to the exe file on linux (RPI4B). I am new in this so i probably doing something wrong, but the output is: 

(env) thorpi@raspberrypi:~/Documents/irms_project/exe $ pyinstaller --name irMedSys --onedir --windowed --icon=icon.ico --add-data "thorpi@raspberrypi:/home/thorpi/Documents/irms_project/env/lib/python3.9/site-packages/customtkinter;customtkinter/" gui_irms.py 
273 INFO: PyInstaller: 5.10.1
273 INFO: Python: 3.9.2
279 INFO: Platform: Linux-6.1.21-v8+-aarch64-with-glibc2.31
281 INFO: wrote /home/thorpi/Documents/irms_project/exe/irMedSys.spec
289 INFO: UPX is not available.
296 INFO: Extending PYTHONPATH with paths
['/home/thorpi/Documents/irms_project/exe']
1127 INFO: Appending 'datas' from .spec
Unable to find "/home/thorpi/Documents/irms_project/exe/thorpi@raspberrypi" when adding binary and data files.

Could somebody help me?

**Comment by Akascape** [comment]
https://github.com/TomSchimansky/CustomTkinter/discussions/939
